scripts/apps/vocabularies/views/vocabulary-config-modal.html

```diff
@@ -60,14 +60,33 @@ <h3 class="modal__heading" ng-show="!vocabulary._id"><span translate>Create</spa
                 <sd-check id="field-text-single" ng-model="vocabulary.field_options.single">{{'Single line' | translate}}</sd-check>
             </div>
 
-            <div class="form__row" ng-show="matchFieldTypeToTab('media-fields', vocabulary.field_type)">
-                <label for="field-media-type" class="form-label form__row-label" translate>Media Type</label>
-                <div class="form__row-item">
-                    <sd-check name="field_type" ng-model="vocabulary.field_options.allowed_types.picture" data-icon="photo">{{'Image' | translate}}</sd-check><sd-check name="field_type" ng-model="vocabulary.field_options.allowed_types.video" data-icon="video">{{'Video' | translate}}</sd-check><sd-check name="field_type" ng-model="vocabulary.field_options.allowed_types.audio" data-icon="audio">{{'Audio' | translate}}</sd-check>
+            <div class="form__row" ng-show="matchFieldTypeToTab('related-content-fields', vocabulary.field_type)">
+                <label for="field-media-type" class="form-label form__row-label" translate>Content Type</label>
+
+                <div class="form__row form__row--inner form__row--flex">
+                    <div class="form__row-item form__row-item--no-grow">
+                        <select class="input-medium" ng-model="vocabulary.field_options.contentType">
+                            <option value="media-gallery" translate>Media gallery</option>
+                            <option value="related-items" translate>Related items</option>
+                        </select>
+                    </div>
+                    <div class="form__row-item form__row-item--no-grow">
+                        <div class="form__row">
+                            <sd-check name="field_type" ng-model="vocabulary.field_options.allowed_types.picture" data-icon="photo">{{'Image' | translate}}</sd-check>
+                            <sd-check name="field_type" ng-model="vocabulary.field_options.allowed_types.video" data-icon="video">{{'Video' | translate}}</sd-check>
+                            <sd-check name="field_type" ng-model="vocabulary.field_options.allowed_types.audio" data-icon="audio">{{'Audio' | translate}}</sd-check>
+                        </div>
+                        <div class="form__row" ng-if="vocabulary.field_options.contentType === 'related-items'">
```

Comment:
afaik picture/video/audio should be here too, and it should be using [cv](https://github.com/superdesk/superdesk-core/blob/master/apps/prepopulate/data_init/vocabularies.json#L112-L128)

Reply:
picture/video/audio are above and are shown in both cases

Reply:
:+1: my bad